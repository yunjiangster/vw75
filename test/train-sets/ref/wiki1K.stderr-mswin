your learning rate is too high, setting it to 1
Num weight bits = 13
learning rate = 1
initial_t = 1
power_t = 0.5
using no cache
Reading datafile = train-sets/wiki1K.dat
num sources = 1
average    since         example     example  current  current  current
loss       last          counter      weight    label  predict features
10.380830  10.380830           3         3.0  unknown   0.0000       37
10.456700  10.532571           6         6.0  unknown   0.0000       13
10.398070  10.327713          11        11.0  unknown   0.0000       31
10.499986  10.601903          22        22.0  unknown   0.0000        1
10.451025  10.402064          44        44.0  unknown   0.0000      165
10.446713  10.442300          87        87.0  unknown   0.0000       28
10.076803  9.706893          174       174.0  unknown   0.0000       16
9.506702   8.936600          348       348.0  unknown   0.0000        1
8.930677   8.354652          696       696.0  unknown   0.0000      142

finished run
number of examples = 1000
weighted example sum = 1000
weighted label sum = 0
average loss = 8.716
best constant = 1
total feature number = 86919
