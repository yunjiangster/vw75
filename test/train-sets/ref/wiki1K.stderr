Num weight bits = 13
learning rate = 1
initial_t = 1
power_t = 0.5
using no cache
Reading datafile = train-sets/wiki1K.dat
num sources = 1
average    since         example     example  current  current  current
loss       last          counter      weight    label  predict features
10.149301  10.149301           1         1.0  unknown   0.0000      732
10.369812  10.590324           2         2.0  unknown   0.0000       27
10.325923  10.282033           4         4.0  unknown   0.0000       53
10.401762  10.477602           8         8.0  unknown   0.0000       60
10.356291  10.310820          16        16.0  unknown   0.0000       26
10.472940  10.589588          32        32.0  unknown   0.0000      125
10.474844  10.476749          64        64.0  unknown   0.0000      313
10.425304  10.375763         128       128.0  unknown   0.0000       50
9.797446   9.169588          256       256.0  unknown   0.0000       33
9.251716   8.705987          512       512.0  unknown   0.0000       26

finished run
number of examples = 1000
weighted example sum = 1000
weighted label sum = 0
average loss = 8.84619
best constant = 1
total feature number = 86919
